static/src/javascripts/projects/common/modules/experiments/tests/frontend-dotcom-rendering-epic.js

```diff
@@ -200,6 +202,26 @@ const frontendDotcomRenderingTest = {
                                     trackingCampaignId,
                                     products
                                 );
+
+                                // If the Epic has custom JS code,
+                                // we need to eval it and call the function
+                                // it defines globally
```

Comment:
Maybe just clarify here that this is a temporary approach that we plan on changing very soon?

Reply:
Yup, done